app/src/state/transcribe.ts

```diff
@@ -127,6 +128,7 @@ export const startTranscription = createAsyncThunk<
             })
           );
         }
+        dispatch(setDisplaySpeakerNames(diarize));
```

Comment:
yeah this is a bit of a hack because we probably want openDocumentFromMemory to clear the editor state?
maybe change the dispatch order?

Reply:
Done